api/mapping/layerselector/request/showfilteredlayerlistrequest.md

```diff
@@ -9,69 +9,53 @@ or selected filter as well as optional paramater, which is open layer list.
 
 Default filter are:
 * newest, this shows 20 newest layers on layerlist
-* stats , this shows only analysable layers on layerlist
-* publishable, this shows only publishable layers on layerlist
+* featuredata, this shows only featuredata layers on layerlist
+
+Bundles registered filters:
+* publishable, this shows only publishable layers on layerlist. Available only then if publisher2 bundle started and user is logged in.
 
 You can use default filters only when filter buttons are showed on layerlist.
 
 Filter layerlist to show only newest layers using the code below:
```

Comment:
I think the examples having false as second parameter should only send one param and have one of them use true with description what it does (opens the flyout?).